modules/core/src/experimental/utils/gpu-grid-aggregation/gpu-grid-aggregator.js

```diff
@@ -357,13 +694,81 @@ export default class GPUGridAggregator {
     this.allAggregationModel = this._getAllAggregationModel(fp64);
   }
 
+  _updateAggregationBuffers(opts, results) {
+    if (!opts.createBufferObjects) {
+      return;
+    }
+    const weights = opts.weights || this.state.weights;
+    for (const id in results) {
+      const {aggregationData, minData, maxData, maxMinData} = results[id];
+      const {needMin, needMax} = weights[id];
+      const calculateMinMax = needMin || needMax;
+      const combineMaxMin = needMin && needMax && weights[id].combineMaxMin;
+      updateBuffer({
+        gl: this.gl,
+        bufferName: 'aggregationBuffer',
+        data: aggregationData,
+        result: results[id]
+      });
+      if (calculateMinMax) {
```

Comment:
I'm surprised lint allows you to nest this deep. Again `calculateMinMax` is not really needed?